src/components/form-attachment/list.vue

```diff
@@ -28,6 +28,7 @@ except according to the terms contained in the LICENSE file.
           <th class="form-attachment-list-type">{{ $t('header.type') }}</th>
           <th class="form-attachment-list-name">{{ $t('header.name') }}</th>
           <th class="form-attachment-list-uploaded">{{ $t('header.uploaded') }}</th>
+          <th class="form-attachment-list-action"></th>
```

Comment:
I have added a wrapper div in form-attachment-list-action with the width of 200px, also changed styling of uploaded column so that it doesn't grow infinitely